add disambiguation page to /stat (#816)
arXiv/arxiv-docs: source/help/stat/index.md

# Stat Disambiguation 

> [Click here](../statistics/index.md) for the help page for the statistics (stat) category for arXiv papers.

Reviewer: ```suggestion
[Click here](../statistics/index.md) to see information about the Statistics (stat) archive.
```
